pkgs/by-name/lu/luarocks-packages-updater/updater.py

```diff
@@ -231,7 +238,8 @@ def generate_pkg_nix(plug: LuaPlugin):
         log.debug("running %s", " ".join(cmd))
 
         output = subprocess.check_output(cmd, text=True)
-        output = "callPackage(" + output.strip() + ") {};\n\n"
+        ## FIXME: luarocks nix command output isn't formatted properly
+        output = "callPackage(\n" + output.strip() + ") {};\n\n"
```

Comment:
@teto I see the package upstream we're using hasn't been updated in a while, but would you be able to make the json formatted using the new formater?

Reply:
calling the formatter on the generated code is a workaround. I've invited you and @GaetanLepage as maintainers of luarock-nix in case you wanna change it.

As for the errors you have, it's easier to spot errors byt launching the program with `--proc=1 --debug DEBUG`.
You can also launch the `luarocks nix --verbose` command yourself and the error is (now) straightforward

Reply:
Thanks, appreciate it! I'll try debugging it some later